### PR TITLE
Remove proxy - created on top of feature/deploy

### PR DESCRIPTION
**Pull request by Yordanofff**

### CODE REVIEW

File: crawler/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-
-#Work with DigitalOceanDatabase
-spring.datasource.username=${DB_USERNAME}
-spring.datasource.password=${DB_PASSWORD}
-spring.datasource.url=jdbc:postgresql://db-postgresql-fra1-77554-do-user-17282102-0.h.db.ondigitalocean.com:25060/defaultdb
+# Works for production and for local development (if ENV are not set - docker-compose-local.yml)
+spring.datasource.username=${DB_USERNAME:root}
+spring.datasource.password=${DB_PASSWORD:root}
+spring.datasource.url=jdbc:postgresql://${POSTGRES_HOST:localhost}:${DB_PORT:5432}/${DB_TABLE_NAME:db_properties}
+#spring.datasource.url=jdbc:postgresql://${POSTGRES_HOST:localhost}:5432/${DB_TABLE_NAME:db-table-name}
```
Comment (emiliyank) [inline crawler/src/main/resources/application.properties]: This can be removed

File: crawler/src/main/resources/templates/attach-pdf.html [hunk 2]
```diff
@@ -46,8 +46,8 @@ <h1>Сравнителни оценки на пазара</h1>
     </div>
     <div class="d-flex justify-content-center">
       <button type="submit" class="btn btn-lg btn-primary me-3">Генерирай оценки</button>
-      <a href="http://198.199.122.215:8080/" class="btn btn-lg btn-info me-3" role="button">Връщане към начало</a>
-      <a href="http://198.199.122.215:8080/download-template" class="btn btn-lg btn-warning" role="button">Свали Word шаблон</a>
+      <a th:href="@{/}" class="btn btn-lg btn-info me-3" role="button">Връщане към начало</a>
```
Comment (emiliyank) [inline crawler/src/main/resources/templates/attach-pdf.html:49]: Very good changes here for removing the hardcoded URL

File: crawler/docker-compose-local.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Locally - use "docker compose -f .\docker-compose-local.yml up -d" to build it
```
Comment (emiliyank) [inline crawler/docker-compose-local.yml:1]: It is good that we have docker-compose for local and prod environments!

**Review by emiliyank — APPROVED** [review]

File: crawler/src/main/resources/application.properties (hunk 1 above)
Comment (Yordanofff) [reply crawler/src/main/resources/application.properties]: I've missed it while debugging. Thanks

**Pull request merged**